v2/pkg/subscraping/sources/hunter/hunter.go

```
@@ -57,10 +57,48 @@ func (s *Source) Run(ctx context.Context, domain string, session *subscraping.Se
+				subdomain := hunterInfo.Domain
+				results <- subscraping.Result{Source: s.Name(), Type: subscraping.Subdomain, Value: subdomain}
+				s.results++
+			}
+		}
+		// get all pages 获取全部页数
+		pages = int(response.Data.Total/1000) + 1
+		//page turning 进行翻页操作
+		for currentPage := 2; currentPage <= pages; currentPage++ {
 			qbase64 := base64.URLEncoding.EncodeToString([]byte(fmt.Sprintf("domain=\"%s\"", domain)))
-			resp, err := session.SimpleGet(ctx, fmt.Sprintf("https://hunter.qianxin.com/openApi/search?api-key=%s&search=%s&page=1&page_size=100&is_web=3", randomApiKey, qbase64))
+			resp, err := session.SimpleGet(ctx, fmt.Sprintf("https://hunter.qianxin.com/openApi/search?api-key=%s&search=%s&page=%s&page_size=100&is_web=3", randomApiKey, qbase64, currentPage))
```

golangci-lint (1.64.8)

printf: fmt.Sprintf format %s has arg currentPage of wrong type int

(govet)